### PR TITLE
Adding a first pass at the day three discovery document.

### PR DESCRIPTION
**Pull request by 1Copenut**
## Purpose
This is the first pass at a Day Three discovery document. We want to work with onboarding teams to discover as much as we can about their project. We would want to write actionable tickets by the end of this discovery session so the project team could move forward with a clear vision.

## Items Included
* General file for discovery meeting
* Template for contact information - this could be considered PII and might need to be private
* Template for note-taking during Day Three discovery meeting(s)

## Goal Items to Define and Capture
* A timeline for this meeting, including logistics (running time, multiple small meetings, one large meeting)
* A way to transform notes into discrete tasks that can be entered into Github and give onboarding team next actions

## Related Ticket
https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/424

### CODE REVIEW

**Comment by 1Copenut** [comment]
@smoonpeyes Love this comment. Just added a line item and some sub-points to support it.

File: Onboarding/day-three-strategy-doc.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+- Copy the sample-product-discovery-questions file into project folder **(TODO: Add URL when PR is approved)**
+
+## Agenda
+
+- Getting set up
+- Discovery
+  - Define points of contact
+  - Ask sample questions
+  - Discuss implementation
+- First break
```
Comment (hpjaj) [inline Onboarding/day-three-strategy-doc.md]: 💯 for turning this into more of a workshop, than a meeting.  I would run with that, and frame it as such, i.e. change the title of things from "meeting" to "workshop", etc.

File: Onboarding/day-three-strategy-doc.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+- Ensure VSP understands the project parameters
+- Learn about the onboarding team's implementation ideas and strategies
+
+### Learn About the Product
+
+**1 hour to 1.75 hours:** This is the VSP team's opportunity to ask questions and learn about the onboarding team's product.
+
+- Establish the points of contact. Fill out the team contacts document.
+- Review questions from the sample-product-discovery-questions doc **(TODO: Add URL when PR is approved)**
+- Is team integrating a new external API at the onset?
```
Comment (hpjaj) [inline Onboarding/day-three-strategy-doc.md]: 👍

File: Onboarding/day-three-strategy-doc.md (hunk 1 above)
Comment (1Copenut) [reply Onboarding/day-three-strategy-doc.md]: 👍

**Pull request merged**